docs/tutorials/multimodal/advanced_topics/tensorrt.ipynb

```diff
@@ -56,6 +56,7 @@
     "    print(f\"tensorrt=={tensorrt.__version__}, onnx=={onnx.__version__}, onnxruntime=={onnxruntime.__version__}\")\n",
     "except ImportError:\n",
     "    !pip install autogluon.multimodal[tests]\n",
+    "    !pip install -U --pre tensorrt\n",
```

Comment:
Thanks for the fix! Can we add a TODO to this line so that we can monitor to have it removed in case the pre-release version breaks in future?

Reply:
It is not best practice to introduce TODO item in external facing doc. I have pin the version while creating a tracking issue to keep it on our radar